docs/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.md

```diff
@@ -411,6 +411,16 @@ A plugin has 2 different ways of injecting a component:
 * to inject a component from a plugin into another plugin's injection zones, use the `injectComponent()` function
 * to specifically inject a component into one of the Content Manager's [predefined injection zones](#using-predefined-injection-zones), use the `injectContentManagerComponent()` function instead
 
+Both the `injectComponent()` and `injectContentManagerComponent()` methods accept the following arguments:
+
+| Argument        | Type   | Description                                                                                                                                                                   |
+| --------------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| first argument  | String | The view where the component is injected
+| second argument | String | The zone where the component is injected
+| third argument  | Object | An object with the following keys:<ul><li>`name` (string) the name of the component</li><li>`Component` (function) A function to returns the content to be injected</li></ul> |
+
+<!-- ? is Component a function or could it be something else? I see inconsistencies between the example in this documentation and the notes I received from devs :thinking-face: ? -->
```

Comment:
Component is a React component so either a function or a class. It cannot be a promise or an async function